test/dotnet/ExpectedOutput/insertBulk.out

```diff
@@ -379,6 +379,244 @@ hello#!#jello
 2#!#2
 #Q#drop table sourceTable
 #Q#drop table destinationTable
+#Q#create table sourceTable(c1 int identity, c2 CHAR(1024))
+#Q# set IDENTITY_INSERT sourceTable ON
+#Q#INSERT INTO sourceTable (c1, c2) SELECT generate_series(1, 1001, 1), 'Foo';
+#Q# set IDENTITY_INSERT sourceTable OFF
+#Q#create table destinationTable(c1 int identity, c2 CHAR(1024), check (c1 < 500))
+#Q#create table sourceTable1(c1 int identity, c2 CHAR(1024))
+#Q# set IDENTITY_INSERT sourceTable1 ON
+#Q#INSERT INTO sourceTable1 (c1, c2) VALUES(1001, 'Foo')
+#Q#INSERT INTO sourceTable1 (c1, c2) SELECT generate_series(1, 1000, 1), 'Foo';
+#Q# set IDENTITY_INSERT sourceTable1 OFF
+#Q#Select count(c1) from sourceTable
+#D#int
+1001
+#Q#select count(c1) from sourceTable1
+#D#int
+1001
+#Q#select count(c1) from destinationTable
+#D#int
+2002
```

Comment:
The exact same test case written in insertBulkErrors.out has this output:
```
#Q#select count(c1) from sourceTable1
#D#int
1001
#Q#select count(c1) from destinationTable
#D#int
0
#Q#select Ident_CURRENT('sourceTable')
#D#decimal
1001
#Q#select Ident_CURRENT('sourceTable1')
#D#decimal
1001
#Q#select Ident_CURRENT('destinationTable')
#D#decimal
1
```
Here we seem to have copied rows for destinationTable but not in insertBulkErrors.out
Please explain and fix it